pycytominer/tests/test_cyto_utils/test_cells.py

```diff
@@ -429,10 +437,102 @@ def test_merge_single_cells_subsample():
         )
 
 
+def test_merge_single_cells_annotate():
+    """
+    Tests SingleCells.merge_single_cells using optional annotate functionality
+    """
+
+    expected_sc_merged_df = annotate(
+        profiles=AP.merge_single_cells(),
+        platemap=PLATEMAP_DF,
+        join_on=["Metadata_well_position", "Metadata_Well"],
+    )
+    sc_merged_df = AP.merge_single_cells(
+        platemap=PLATEMAP_DF, join_on=["Metadata_well_position", "Metadata_Well"]
+    )
+
+    pd.testing.assert_frame_equal(sc_merged_df, expected_sc_merged_df)
+
+
+def test_merge_single_cells_cytominer_database_test_file():
+    """
+    Tests SingleCells.merge_single_cells using cytominer-database test file
+    """
+
+    # read test file based on cytominer-database exports
+    sql_path = os.path.join(
+        os.path.dirname(__file__),
+        "../test_data/cytominer_database_example_data/test_SQ00014613.sqlite",
+    )
+    sql_url = f"sqlite:///{sql_path}"
+
+    # build SingleCells from database
+    sc_p = SingleCells(
+        sql_url,
+        strata=["Image_Metadata_Plate", "Image_Metadata_Well"],
+        image_cols=["TableNumber", "ImageNumber"],
+    )
+
+    # gather base merge_single_cells df
+    merged_sc = sc_p.merge_single_cells()
+
+    # test csv output from merge_single_cells
+    result_file = sc_p.merge_single_cells(
+        sc_output_file=os.path.join(
```

Comment:
please switch from `os` to `pathlib`

I'm not sure how much `os` legacy code exists, but we currently much prefer pathlib as a direct replacement to os.

Reply:
Understood, will do, apologies about this.